src/ocean/text/convert/Formatter.d

```diff
@@ -237,7 +237,7 @@ public bool sformat (Args...) (FormatterSink sink, cstring fmt, Args args)
 deprecated("Use the overload which accepts a `FormatterSink` instead")
 public bool sformat (Args...) (Sink sink, cstring fmt, Args args)
 {
-    return sformat!(Args)((cstring s) { sink(s); }, fmt, args);
+    return sformat((cstring s) { sink(s); }, fmt, args);
```

Comment:
Huh, so it's just a quirk of the compiler that the changed code has the same effect but doesn't trigger the bug?